.github/workflows/bootstrap-secrets.yml

```
@@ -7,15 +7,23 @@ on:
   bootstrap:
     runs-on: ubuntu-latest
+    env:
+      STRIPE_SECRET_KEY: ${{ secrets.STRIPE_SECRET_KEY }}
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      DB_HOST: ${{ secrets.DB_HOST }}
+      JWT_SECRET: ${{ secrets.JWT_SECRET }}
     steps:
       - uses: actions/checkout@v4
 
+      - name: Sync secrets from HashiCorp Vault (optional)
+        if: ${{ secrets.VAULT_ADDR != '' && secrets.VAULT_TOKEN != '' && secrets.VAULT_SECRETS != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)